inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java

```diff
@@ -141,6 +141,8 @@ public class InlongConstants {
      */
     public static final String DATAFLOW = "dataflow";
 
+    public static final String STREAM_CONFIG_STATUS = "stream.config.status";
```

Comment:
Please rename, this information is only used to determine whether scheduling is enabled, and the naming of this parameter does not match its actual meaning